gpu/gpu_info_cpu.c

```diff
@@ -26,6 +26,7 @@ void cpu_check_ram(mem_info_t *resp) {
   if (sysinfo(&info) != 0) {
     resp->err = strdup(strerror(errno));
   } else {
+    resp->count = 1;
```

Comment:
For symmetry and to keep the test passing on windows, you should add the same at around line 11.